loculus_values/values.yaml

```diff
@@ -60,6 +60,12 @@ sequenceFlagging:
     organization: pathoplexus
     repository: curation_reports
     issueTemplate: test-sequence-metadata-issue.md
+siloImport:
+  siloTimeoutSeconds: 3600
+  hardRefreshIntervalSeconds: 100_000 # ~27 hours
```

Comment:
Quick thought: if we update a value in the database via db surgery will this be picked up by SILO without the hard refresh?